src/client/routes/Home/index.tsx

```diff
@@ -86,7 +87,7 @@ export const Home = () => {
 
   return (
     <StyledViewContainer>
-      <HeaderCard header="Dashboard" items={summaryCardItems} variant="secondary" cardSize="small" />
```

Comment:
Is this change approved and modified in figma @pentcle ?

Reply:
we discussed this last week and the cards are in the feedback column
how does the design team take these cards to adapt figma?

Reply:
will put these back, although i would strongly recommend removing the word dashboard, as these are more holdings summary panels (eg value of holdings) – a dashboard tends to be something a bit more complex